folly/experimental/io/test/Makefile.am

```diff
@@ -1,12 +1,5 @@
 ACLOCAL_AMFLAGS = -I m4
```

Comment:
Seeing this after the rebase, this entire file should just be removed, as it no longer does anything useful.